docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 # To boot the app run the following:
-# docker-compose run auto-gpt
-# NOTE: Version 3.9 requires at least docker-compose version 1.29.0 !
+# docker compose run auto-gpt
+# NOTE: Version 3.9 requires at least Docker Compose version 1.29.0 !
```

Comment:
Since `compose` is now a sub-command of `docker`, I think this should be a minimum `Docker` version requirement?

Reply:
https://www.howtogeek.com/devops/how-to-upgrade-to-docker-compose-v2/

Reply:
On it, one sec